samples/python/agents/langgraph/__main__.py

```diff
@@ -19,8 +19,23 @@
 def main(host, port):
     """Starts the Currency Agent server."""
     try:
-        if not os.getenv("GOOGLE_API_KEY"):
-            raise MissingAPIKeyError("GOOGLE_API_KEY environment variable not set.")
+        if (
+            not os.getenv("GOOGLE_API_KEY")
```

Comment:
Can't we do it like below ?

`def check_environment_variables_any(variable_names):
  if not any(os.getenv(var) is not None for var in variable_names):
    raise EnvironmentError(f"None of the required environment variables are set: {', '.join(variable_names)}")` 

variable_names is a set of environment variables supported by the protocol